hotshot-stake-table/src/mt_based.rs

```diff
@@ -9,10 +9,10 @@
 mod config;
 mod internal;
 
+use alloy::primitives::{U256, U512};
```

Comment:
nit: I feel like we should deprecate this module.

Reply:
sure, maybe in future PR.